compiler/test/test_end_to_end.re

```diff
@@ -922,7 +922,7 @@ let gc = [
     "(1, 2)",
   ),
   tgcfile("fib_gc_err", 1024, "fib-gc", "Out of memory"),
-  tgcfile("fib_gc", 3072, "fib-gc", "832040"),
+  tgcfile("fib_gc", 4000, "fib-gc", "832040"),
```

Comment:
What does it mean that this goes up to 4000?

Reply:
It's just that this doesn't generate the same code anymore (because of the tail call instruction) and that changed the amount of memory this uses. It definitely uses _less_ memory than before, but the way we test this is currently broken (see #504), and the way our testing works makes this number a little arbitrary.

Reply:
I did a binary search to find how much memory spanned—that's a better term for what this is—happed and updated it.